lib/stitches/generator_files/app/controllers/api/api_controller.rb

```diff
@@ -1,10 +1,20 @@
 class Api::ApiController < ActionController::Base
   include Stitches::Deprecation
+  #
+  # The order of the rescue_from blocks is important - ActiveRecord::RecordNotFound must come after StandardError,
+  # otherwise "not_found" errors will get rescued in the StandardError block.
+  # See the documentation for rescue_from for further explanation:
+  # https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from
+  # Specifically, this part: "Handlers are inherited. They are searched from right to left, from bottom to top, and up
+  # the hierarchy."
+  #
+  rescue_from StandardError do |exception|
+    Bugsnag.notify(exception)
+    render json: { errors: Stitches::Errors.from_exception(exception) }, status: :internal_server_error
+  end
+
   rescue_from ActiveRecord::RecordNotFound do |exception|
-    respond_to do |type|
-      type.json { render json: { errors: Stitches::Errors.new([ Stitches::Error.new(code: "not_found", message: exception.message) ]) }, status: 404 }
-      type.all  { render :nothing => true, :status => 404 }
-    end
+    render json: { errors: Stitches::Errors.new([ Stitches::Error.new(code: "not_found", message: exception.message) ]) }, status: :not_found
```

Comment:
Why not use `from_exception` here?

Reply:
Good question! The `. from_exception` method doesn't allow us to set a custom error code, since it [derives the code from the exception class](https://github.com/stitchfix/stitches/blob/1d2c1e5d90986de476cb77ac4ed78e98a5c99d4a/lib/stitches/errors.rb#L61).

In this case, that would make the error code `active_record/record_not_found`, which I don't think is what we want, since we've been consistently using `not_found`.

Reply:
Discussed in slack, and we're going to go ahead and make the change to use `.from_exception` for consistency with how other exceptions are handled. Since this is a generator, it won't break any existing services that are currently using `Stitches::Errors.new` to set the custom `not_found` code.